apps/frontend/src/app/(app)/space/this-week/page.tsx

```diff
@@ -1,5 +1,137 @@
+          <div className="flex items-center justify-between gap-4">
+            <button onClick={() => console.log("left")} className="p-2">
+              <LeftChevron />
+            </button>
+            <button onClick={() => console.log("right")} className="p-2">
+              <RightChevron />
+            </button>
+          </div>
+        </div>
+        <div className="flex w-full max-w-screen-xl gap-8">
+          <div className="flex flex-col flex-1 gap-4 group/section">
+            <div className="flex items-center gap-2 text-xl text-foreground">
+              <Icon
+                icon="material-symbols:circle-outline"
+                style={{ fontSize: "20px" }}
+              />
+              <h2>to do</h2>
+            </div>
+            <div className="flex flex-col text-left gap-1 p-4 border border-border rounded-lg hover-bg group">
+              <div className="flex justify-between text-foreground">
+                <div className="w-full flex items-start gap-2">
+                  <Icon
+                    icon="ri:github-fill"
+                    style={{ fontSize: "18px" }}
+                    className="mt-0.5"
+                  />
+                  <p>title</p>
+                </div>
+                <div className="text-secondary-foreground text-xs">
+                  <button className="invisible group-hover:visible hover-text">
+                    edit
+                  </button>
+                </div>
+              </div>
+              <div className="ml-[18px] pl-2 text-xs">
+                <p>description</p>
+              </div>
+            </div>
+            <button className="invisible flex flex-col text-left text-sm gap-1 p-4 rounded-lg hover-bg group-hover/section:visible">
+              <div className="flex items-center gap-2">
+                <Icon icon="ic:round-plus" style={{ fontSize: "18px" }} />
+                <p>New item</p>
+              </div>
+            </button>
+          </div>
+          <div className="flex flex-col flex-1 gap-4 group/section">
+            <div className="flex items-center gap-2 text-xl text-foreground">
+              <Icon icon="carbon:circle-dash" style={{ fontSize: "20px" }} />
+              <h2>in progress</h2>
+            </div>
+            <div className="flex flex-col text-left gap-1 p-4 border border-border rounded-lg hover-bg group">
+              <div className="flex justify-between text-foreground">
+                <div className="w-full flex items-start gap-2">
+                  <Icon
+                    icon="ri:github-fill"
+                    style={{ fontSize: "18px" }}
+                    className="mt-0.5"
+                  />
+                  <p>title</p>
+                </div>
+                <div className="text-secondary-foreground text-xs">
+                  <button className="invisible group-hover:visible hover-text">
+                    edit
+                  </button>
+                </div>
```

Comment:
A lot of repetition is happening, Create a array of objects and then render the elements using map function. You can read about the DRY principle as well.

Reply:
sure, i just did it static to see how it looks
now i will do it with real data